web/src/components/navigation.js

```diff
@@ -38,27 +39,50 @@ const NavigationLink = styled.a`
 
 
 class Navigation extends Component {
+ 
   render() {
     return (
+      <Router>
       <NavigationList>
         <NavigationListItem>
-          <NavigationLink href="/">Dashboards</NavigationLink>
+            <NavigationLink href="/">
+                <Link to="/">Dashboards</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/coops">Co-Ops</NavigationLink>
+            <NavigationLink href="/coops">
+                <Link to="/coops/">Co-Ops</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/members">Members</NavigationLink>
+            <NavigationLink href="/members">
+                <Link to="/members/">Members</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/transactions">Transactions</NavigationLink>
+            <NavigationLink href="/transactions">
+                <Link to="/transactions/">Transactions</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/loans">Loans</NavigationLink>
+            <NavigationLink href="/loans">
+                <Link to="/loans/">Loans</Link>
+            </NavigationLink>
           </NavigationListItem>
       </NavigationList>
+
+      <Route path="/" exact component={this.Index} />
```

Comment:
so I think I told you the wrong thing before, but I think we should keep this whole `<Router>` section in App.js in place of the `{content}` line. Then we can also move the `import { BrowserRouter as Router, Route } from "react-router-dom";` import to App.js and have the components load in the content area.